### DISCUSSION TITLE
blog/cs/2024-08-15-About-Message-Queue/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#21] [Announcements]
# blog/cs/2024-08-15-About-Message-Queue/

moonshot

https://mingdodev.github.io/blog/cs/2024-08-15-About-Message-Queue/

**Comment by hwanheejung** [comment]
Kafka가 왜 대규모 스트리밍에 적합한가요?? 어떻게 동작하길래..
